input/papers/2016/2016arXiv160908671H/tev-000095.yaml

```diff
@@ -0,0 +1,26 @@
+paper_id: 2016arXiv160908671H
+source_id: 95
+
+data:
+  livetime: 13.4 # year 2012
+#  livetime: 65.3 year 2011, earlier observations have already been published
+#  significane: 3, 5, 7, 9 mentioned in paper fig 4
+
+pos:
+  ra: {val: 17.226} # nominal centre of the SNR, see tab 1, what about centred 17h13m25.2s see fig 2?
+  dec: {val: -39.76} # nominal centre of the SNR, see tab 1, wht about centred -39d46m15.6s see fig 2?
```

Comment:
The text (paragraph 3) mentions this position as the fit result: 
> The resulting centre point is R.A.: 17h13m25.2s, Dec.: −39d46m15.6s

I would suggest to either put that position, or to put the position from SIMBAD for this SNR:
http://simbad.u-strasbg.fr/simbad/sim-id?Ident=+SNR+G347.3-00.5&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id
For the comment: put it on the line before and don't duplicate the text.